webview-ui/src/components/settings/__tests__/ApiConfigManager.spec.tsx

```diff
@@ -89,6 +89,21 @@ vitest.mock("@/components/ui", () => ({
 			{children}
 		</option>
 	),
+	SearchableSelect: ({ value, onValueChange, options, placeholder, "data-testid": dataTestId }: any) => (
+		<select
+			value={value}
+			onChange={(e) => {
+				if (onValueChange) onValueChange(e.target.value)
+			}}
+			data-testid={dataTestId || "select-component"}>
+			<option value="">{placeholder || "Select..."}</option>
```

Comment:
Should internationalize this Select... (or do we already have a string for it somewhere?)